crates/common_utils/src/pii.rs

```diff
@@ -87,26 +99,77 @@ where
     }
 }
 
-/// Email address
+/// Strategy for masking Email
 #[derive(Debug)]
-pub struct Email;
+pub struct EmailStrategy;
```

Comment:
You should be able to make this private, since no other module would need it.

```suggestion
struct EmailStrategy;
```

Reply:
`Queryable` requires EmailStrategy to be public, since it is inside public interface
> error[E0446]: private type `EmailStrategy` in public interface

Reply:
Alright then, you can ignore my comment. Marking this conversation resolved.